### PR TITLE
Updates according to comments by Juanjo Hierro and Daniel Villalba

### PR DESCRIPTION
**Pull request by jason-fox**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,58 +1,63 @@
...
-FIWARE **NGSI** is a universal standard for context data management used to facilitate the development of Smart
-Solutions. The FIWARE ecosystem means a straight-forward business opportunity for IoT products - It is simple, any
-device which is capable of directly or indirectly communicating using the NSGI protocol can be plugged into literally
-any FIWARE-based smart solution. This eliminates vendor lock-in and allows new entrants to the IoT device market to
-ensure their compatibility with existing solutions apriori.
+The [FIWARE Marketplace](https://marketplace.fiware.org) serves the purpose of globally disseminating existing
+commercial offerings around FIWARE. It is a global one-stop shop that gives visibility to a wide range of Powered by
+FIWARE solutions/platforms, FIWARE-ready solutions and IoT devices as well as FIWARE related training/coaching or
+consultancy, integration and support services. Entries in the FIWARE Marketplace have been validated by the FIWARE
+Foundation or FIWARE iHubs which meet the requirements for performing validation of FIWARE Marketplace entries
```
Comment (flopezag) [inline docs/index.md]: finished sentence without dot.

File: docs/solution/guideline.md [hunk 2]
```diff
@@ -63,9 +63,10 @@ In order to proceed with the validation of a _"Powered by FIWARE"_ Solution vali
    under points 2 and 3 (either a. or b.). Note that there is a **strong recommendation** that common data models should
    be used where they exist, but are not mandatory for novel use-cases where a common data model has not yet been
    created. In the later case, for the new data model or additional attributes for an existing data model should be
-   extending common vocabulary where possible and follow the usual rules for data models (e.g attributes in English,
-   camel case etc.). For common future use, in the ideal case the candidate should be encouraged to submit a new common
-   data model or add additional fields as necessary.
+   extending common vocabulary where possible and follow the usual
+   [rules for data models](https://fiware-datamodels.readthedocs.io/en/latest/guidelines/index.html) (e.g attributes in
+   English, camel case etc.). For common future use, in the ideal case the candidate should be encouraged to submit a
```
Comment (flopezag) [inline docs/solution/guideline.md]: , in the ideal case, the candidates (maybe?)

Comment (jason-fox) [reply docs/solution/guideline.md]: Fixed 1f361208c

File: docs/index.md (hunk 1 above)
Comment (jason-fox) [reply docs/index.md]: Fixed 1f361208c

Comment (flopezag) [reply docs/index.md]: LGTM

File: docs/solution/guideline.md (hunk 2 above)
Comment (flopezag) [reply docs/solution/guideline.md]: LGTM

**Pull request merged**